src/test/scala/uk/gov/nationalarchives/tdr/api/db/repository/FileMetadataRepositorySpec.scala

```diff
@@ -1,19 +1,43 @@
 package uk.gov.nationalarchives.tdr.api.db.repository
 
+import java.sql.{PreparedStatement, Timestamp}
+import java.time.Instant
 import java.util.UUID
 
+import org.scalatest.BeforeAndAfterEach
 import org.scalatest.concurrent.ScalaFutures
 import org.scalatest.flatspec.AnyFlatSpec
 import org.scalatest.matchers.should.Matchers
+import org.scalatest.time.{Seconds, Span}
 import uk.gov.nationalarchives.tdr.api.db.DbConnection
+import uk.gov.nationalarchives.tdr.api.graphql.fields.FileMetadataFields.SHA256ServerSideChecksum
 import uk.gov.nationalarchives.tdr.api.utils.TestUtils
 import uk.gov.nationalarchives.tdr.api.utils.TestUtils._
-import uk.gov.nationalarchives.tdr.api.graphql.fields.FileMetadataFields.SHA256ServerSideChecksum
+import uk.gov.nationalarchives.Tables.FilemetadataRow
+
 
 
-class FileMetadataRepositorySpec extends AnyFlatSpec with ScalaFutures with Matchers {
+class FileMetadataRepositorySpec extends AnyFlatSpec with ScalaFutures with Matchers with BeforeAndAfterEach {
+
+  override def beforeEach(): Unit = {
+    val connection = DbConnection.db.source.createConnection()
+    val psFileMetadata = connection.prepareStatement("delete from FileMetadata")
+    val psFileProperty = connection.prepareStatement("delete from FileProperty")
+    psFileMetadata.execute()
+    psFileProperty.execute()
+  }
 
   implicit val ec: scala.concurrent.ExecutionContext = scala.concurrent.ExecutionContext.global
+  override implicit val patienceConfig: PatienceConfig = PatienceConfig(timeout = Span(2, Seconds))
+
+  def getFileChecksumMatches(fileId: UUID): Boolean = {
```

Comment:
Can this be a `private def` as not used outside the class?

Reply:
Yeah, it can, I've made it private.